include/alpaka/mem/buf/cuda/Copy.hpp

```diff
@@ -600,7 +599,7 @@ namespace alpaka
                                 static_cast<std::size_t>(extentDepth));
                         cudaMemCpy3DParms.kind = task.m_cudaMemCpyKind;
 
-                        return cudaMemCpy3DParms;
+                        return cudaMemCpy3DParams;
```

Comment:
You should not fix the NVIDIA CUDA spelling error here ;-) The variable is also `Parms`